odd-platform-ui/src/components/shared/AppToolbar/AppToolbar.tsx

```diff
@@ -60,6 +66,10 @@ const AppToolbar: React.FC<AppToolbarProps> = ({
     fetchIdentity();
   }, []);
 
+  React.useEffect(() => {
+    fetchAppInfo();
+  }, []);
```

Comment:
I think you can call both functions (fetchIdentity and fetchAppInfo) in the one useEffect, because we haven't special deps now.

Reply:
done